blocks/init/src/Blocks/components/modal/modal-style.scss

```diff
@@ -0,0 +1,88 @@
+.modal {
+	$this: &;
+
+	--modal-scoped-icon-size: 1.5rem;
+
+	display: none;
+
+	&.is-open {
+		display: block;
+	}
+
+	&__overlay {
+		position: fixed;
+		top: 0;
+		left: 0;
+		right: 0;
+		bottom: 0;
+		z-index: 999;
+		background-color: rgba(0, 0, 0, 0.5);
```

Comment:
never use colors direct like this. you can use only what you have in the global manifest.json

https://infinum.github.io/eightshift-docs/docs/basics/blocks-styles#global-variables